mkdocs.yml

```diff
@@ -63,3 +63,5 @@ markdown_extensions:
   - pymdownx.inlinehilite
   - pymdownx.snippets
   - pymdownx.superfences
+extra_javascript:
+  - cql_examples.csv
```

Comment:
Don't know if you need this one, since you're generating the options statically (not fetching them w/ JS).